### ISSUE TITLE
Verify email page

### ISSUE THREAD

**Issue by Krishi1211** [Sanskriti1102/CourseGuard#12]
https://www.figma.com/design/DWokUT7bGtMrDVSyj7bp7l/Digiaccel---E-Learning-Platform-(Community)?node-id=5734-14661&t=NxXBNrYk4gzODMg9-1

refer 9,10,11 page for verification of email sent in previous page

**Comment by JainishParekh** [comment]
I wanted to contribute to this issue. I am also contributor in wob can I start working on this issue ?

**Comment by Krishi1211** [comment]
assigned

**Comment by Krishi1211** [comment]
Assigned

**Comment by JainishParekh** [comment]
Thank you for assigning me the issue I will start working on it.

**Comment by JainishParekh** [comment]
@Krishi1211 I just wanted to know which type of routing I should use. Currently, I started with app roututer Can I continue with the same?

**Comment by Krishi1211** [comment]
Yes u can continue with the same

**Comment by Krishi1211** [comment]
Hii @JainishParekh any update on the work?

**Comment by JainishParekh** [comment]
Yes, sorry for not being active. I have started working and done with ui part I was mostly confused about how to connect it with other screens and also about state management. I got little stuck there but by tomorrow I will atleast push static ui.

**Comment by Krishi1211** [comment]
Great

**Comment by JainishParekh** [comment]
When I fetched and tried to build, I am facing an issue of not getting theam in global CSS. Following is the error that I am trying to debug:

ReferenceError: defaultTheme is not defined
Import trace for requested module:
./app/globals.css.webpack[javascript/auto]!=!./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[13].oneOf[12].use[2]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[13].oneOf[12].use[3]!./app/globals.css
./app/globals.css

I tried to resolve where I understood that this issue was related to some configuration of theaming. Can anyone please help me?

**Comment by Krishi1211** [comment]
can you share the ss?

**Comment by JainishParekh** [comment]
![image](https://github.com/user-attachments/assets/dc071c07-1e31-40e2-81df-f079116b8b91)

I cleared the cache then tried to reinstall all the dependecies but still facing this error.

Was not able to contact due to some precomitents. Sorry for reponding late

**Comment by JainishParekh** [comment]
Also, this error is mostly due to the tailwind.config.js file having the following code:

```
      minHeight: {
        ...defaultTheme.height,
      },
      minWidth: {
        ...defaultTheme.width,
      },
```

**Comment by JainishParekh** [comment]
After removing the following line build got complete :

![image](https://github.com/user-attachments/assets/68a72484-f551-4cf7-9f7a-b1b8fdacaea2)

**Comment by Krishi1211** [comment]
Great you can push this

**Comment by JainishParekh** [comment]
I wanted to inform you about some recent changes I made and seek your guidance regarding a few issues I've encountered.

Here are the updates I made:
- **Tailwind CSS Config**: I commented out the lines related to `minHeight` and `minWidth`.
- **Prettier Config**: I updated the `prettier.config.js` to include:
  ```javascript
module.exports = {
  plugins: ["prettier-plugin-tailwindcss"],
  trailingComma: "es5",
  tabWidth: 2,
  printWidth: 120,
  semi: false,
  "prettier/prettier": [
    "error",
    {
      endOfLine: "auto",
    },
  ],
}
  ```
- **Verify Email Page**: I created the verify email page.

Due to the Prettier update, there are quite a few changes across multiple files because the build wasn’t working without those updates. Unfortunately, I made an error with my commit messages and didn’t label them as clearly as I should have. 

I’ve raised a [PR here](https://github.com/Sanskriti1102/CourseGuard/pull/26). However, Vercel bot has also raised some issues that I’m not sure how to resolve, and there are still a few things I’m trying to work through.

Could you kindly review the PR and let me know which changes are acceptable? I would also really appreciate any guidance on resolving the Vercel bot issues.

**Comment by JainishParekh** [comment]
I wanted to inform you about some recent changes I made and seek your guidance regarding a few issues I've encountered.

Here are the updates I made:
- **Tailwind CSS Config**: I commented out the lines related to `minHeight` and `minWidth`.
- **Prettier Config**: I updated the `prettier.config.js` to include:
  ```javascript
  module.exports = {
  plugins: ["prettier-plugin-tailwindcss"],
  trailingComma: "es5",
  tabWidth: 2,
  printWidth: 120,
  semi: false,
  "prettier/prettier": [
    "error",
    {
      endOfLine: "auto",
    },
  ],
}
  ```
- **Verify Email Page**: I created the verify email page.

Due to the Prettier update, there are quite a few changes across multiple files because the build wasn’t working without those updates. Unfortunately, I made an error with my commit messages and didn’t label them as clearly as I should have. 

I’ve raised a [PR here](https://github.com/Sanskriti1102/CourseGuard/pull/26). However, Vercel bot has also raised some issues that I’m not sure how to resolve, and there are still a few things I’m trying to work through.

Could you kindly review the PR and let me know which changes are acceptable? I would also really appreciate any guidance on resolving the Vercel bot issues.

**Comment by JainishParekh** [comment]
I wanted to inform you about some recent changes I made and seek your guidance regarding a few issues I've encountered.

Here are the updates I made:

**Tailwind CSS Config**: I commented out the lines related to `minHeight` and `minWidth`.

**Prettier Config**: I updated the `prettier.config.js` to include:

  ```javascript
  module.exports = {
  plugins: ["prettier-plugin-tailwindcss"],
  trailingComma: "es5",
  tabWidth: 2,
  printWidth: 120,
  semi: false,
  "prettier/prettier": [
    "error",
    {
      endOfLine: "auto",
    },
  ],
}
  ```
  
**Verify Email Page**: I created the verify email page.

Due to the Prettier update, there are quite a few changes across multiple files because the build wasn’t working without those updates. Unfortunately, I made an error with my commit messages and didn’t label them as clearly as I should have. 

I’ve raised a [PR here](https://github.com/Sanskriti1102/CourseGuard/pull/26). However, Vercel bot has also raised some issues that I’m not sure how to resolve, and there are still a few things I’m trying to work through.

Could you kindly review the PR and let me know which changes are acceptable? I would also really appreciate any guidance on resolving the Vercel bot issues.
